src/components/Navigation/Navigation.js

```diff
@@ -26,9 +30,20 @@ const Navigation = ({ setShowMobileNav }) => {
         </button>
       </span>
       <span className="padding-sm secondary__btn-login">
-        <button className="secondary__btn ">
-          <i className="fa-regular fa-user secondary__btn-icon"></i>SIGN IN
-        </button>
+        {isLoggedIn ? (
+          <Link to="/logout">
+            <button className="secondary__btn ">
+              <i className="fa-regular fa-user secondary__btn-icon"></i>LOG OUT
+            </button>
+          </Link>
+        ) : (
+          <Link to="/login">
+            {" "}
```

Comment:
Remove these, it's annoying how it gets added. Unless you added them yourself. 😅

Reply:
Removed this